.gitignore

```diff
@@ -187,6 +187,7 @@ build_android
 build_ios
 /build_*
 .build_debug/*
+.build_profile/*
```

Comment:
this is not really required for this PR, but it's minor a one line change - if anyone insists I'll split it out in a separate PR.

Reply:
This isn't specific to our fuser, so I wonder if it would make more sense to separate this from our develop branch.

Reply:
Fair enough, I reverted this change and I'll create an upstream PR instead